backend/spec/rails_helper.rb

```diff
@@ -21,7 +21,7 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-Dir[Rails.root.join('spec/support/**')].sort.each { |f| require f }
```

Comment:
I had to change this line to use shared examples, I don't know if I could do that. So let me know if it's corret.

Reply:
nice!